dagmc_stats/dagmc_stats.py

```diff
@@ -297,3 +316,172 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_tri_vert_data(my_core, native_ranges):
+    """Build a numpy strcutured array to store triangle and vertex related
+    data in the form of triangle entity handle | vertex entity handle
+    | angle connected to the vertex | side length of side opposite to
+    vertex in the triangle
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    native_ranges : a dictionary containing ranges for each native type
+    in the file (VERTEX, TRIANGLE, ENTITYSET)
+
+    outputs
+    -------
+    tri_vert_data : a numpy structured array that stores the triangle
+    and vertex related data
+    all_verts : (list) all the vertices that are connected to
+    triangle in the geometry
+    """
+    all_tris = native_ranges[types.MBTRI]
+    all_verts = []
+
+    tri_vert_struct = np.dtype({'names': ['tri', 'vert', 'angle',
```

Comment:
Maybe this `dtype` should be defined at global scope in case you want to use it elsewhere.  You don't currently use it elsewhere, but the structured array is used elsewhere.  I can imagine opportunities to invoke this `dtype` in the future in other methods.